style.css

```diff
@@ -43,8 +38,84 @@ footer ul {
   padding-left: 0;
 }
 
-.menu-bar ul {
+.menu-bar {
+  font-family: inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  color: #fff5e1;
+  font-size: 36px;
+  padding-left: 24px;
+  margin-top: 19px;
+  margin-bottom: 19px;
+  position: fixed;
+  top: -1.2rem;
+  background-color: #212121;
+  width: 100%;
+  height: 100vh;
+
+  /* display: flex; */
+  flex-direction: column;
+  align-items: center;
+  padding-top: 10rem;
+  transition: all ease-in-out 0.2s;
   display: none;
+  z-index: 1;
+}
+
+.active {
+  display: block;
+  transition: all ease-in-out 0.2s;
+}
+
+.menu-bar ul {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  gap: 1rem;
+
+  /* padding: 0 0 0 6rem; */
+  padding: 0;
+  margin-top: -87px;
+}
```

Comment:
- [ ] Good work implementing part of the requested changes. please add padding to the right side of the navigation menu  to complete this milestone. 
![menu](https://github.com/ClarenceAtim/Portfolio/assets/99656687/cecf00a9-6bce-4d86-9356-b1c88460be99)